android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java

```diff
@@ -1427,7 +1428,13 @@ private Facts getEntries(String[] address, JSONObject object) throws JSONExcepti
                     formObject.put(RuleConstant.IS_RULE_CHECK, true);
                     formObject.put(RuleConstant.STEP, formObject.getString(RuleConstant.STEP));
 
-                    result.asMap().putAll(getValueFromAddressCore(formObject).asMap());
+
+                    Facts resultFacts = getValueFromAddressCore(formObject);
```

Comment:
So this fixes an issue where comparison values were not available

Reply:
Is there a way we can make a generic function for this?
```
for (Map.Entry<String,Object> factEntry: resultFacts.asMap().entrySet()) {
                        result.put(factEntry.getKey(),factEntry.getValue());
                    }
```
I'm sure it is used severally in the code